test/plugins/builds.test.js

```diff
@@ -235,7 +235,10 @@ describe('build plugin test', () => {
                 scmRepo,
                 admins: { foo: true },
                 sync: sinon.stub().resolves(),
-                syncPR: sinon.stub().resolves()
+                syncPR: sinon.stub().resolves(),
+                admin: Promise.resolve({
```

Comment:
can probably do 
```
admin: sinon.stub().resolves({
    username: 'foo'
})
```
for consistency

Reply:
Let me add more information. `admin` is not a function but a variable, so applying sinon here will be error. `builds.secrets` is also variable, and it applies `Promise`.  The `then` function appears only in case of function. In case of variable, `then` will be undefined, and test failed.

Reply:
ah I understand
thanks for explaining 🙇‍♀️